lib/eth/EthRpc.js

```diff
@@ -169,33 +169,9 @@ class EthRPC {
     return this.estimateGasPrice(nBlocks);
   }
 
-  async estimateGasPrice({ nBlocks = 4 } = {}) {
-    const bestBlock = await this.web3.eth.getBlockNumber();
-    const gasPrices = [];
-    for (let i = 0; i < nBlocks; i++) {
-      const block = await this.web3.eth.getBlock(bestBlock - i);
```

Comment:
`getBlock(num, true) would provide all transactions`

Reply:
updated to that